src/plugins/chart_expressions/expression_metric/common/types/expression_functions.ts

```diff
@@ -23,6 +23,8 @@ export interface MetricArguments {
   showLabels: boolean;
   palette?: PaletteOutput<CustomPaletteState>;
   font: Style;
+  labelFont: Style;
+  labelPosition: 'bottom' | 'top';
```

Comment:
Could you, please, create the const object, which will contain these options? 

It is better to use such a pattern:
https://github.com/elastic/kibana/blob/f49221511c3963e9352c60218ccefcfc5a8a93b0/x-pack/plugins/lens/common/constants.ts#L20
https://github.com/elastic/kibana/blob/f49221511c3963e9352c60218ccefcfc5a8a93b0/x-pack/plugins/lens/common/types.ts#L81

Later, we'll be able to export type from this plugin and mark the current plugin as optional in `Lens`, for example.

Reply:
done